server/requeststojointeamhandler.go

```diff
@@ -0,0 +1,93 @@
+package server
+
+import (
+	"database/sql"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/fluidkeys/api/datastore"
+	"github.com/fluidkeys/api/v1structs"
+	"github.com/fluidkeys/crypto/openpgp"
+	"github.com/gofrs/uuid"
+	"github.com/gorilla/mux"
+)
+
+func listRequestsToJoinTeamHandler(w http.ResponseWriter, r *http.Request) {
+	apparentSignerKey, err := getAuthorizedUserPublicKey(r)
```

Comment:
i think in this case `apparentSignerKey` is a bit misleading because they aren't signing the roster in this request. How about `requesterKey` ?